NetCoreRestApi/DataLayer/Converters/CategoryEntityConverter.cs

```diff
@@ -0,0 +1,39 @@
+﻿using DataLayer.Models;
+using Common.Converter;
+using DataLayer.Entities;
+
+namespace DataLayer.Converters
+{
+    public class CategoryEntityConverter : IConverter<CategoryEntity, CategoryModel>
+    {
+        public CategoryModel ConvertFrom(CategoryEntity categoryEntity)
+        {
+            if (categoryEntity == null)
+            {
+                return null;
+            }
+            var categoryModel = new CategoryModel()
+            {
+                ID = categoryEntity.ID,
+                Name = categoryEntity.Name,
+                Description = categoryEntity.Description
+            };
+            return categoryModel;
+        }
+
+        public CategoryEntity ConvertTo(CategoryModel categoryModel)
```

Comment:
the same we convert entity to model. so it should accept entity

Reply:
done